tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,23 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1020","The cmdlet 'Get-AzVmssVMRunCommand' no longer has output type 'Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20210701.IVirtualMachineRunCommand'.","Make cmdlet 'Get-AzVmssVMRunCommand' return type 'Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20210701.IVirtualMachineRunCommand'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","2000","The cmdlet 'Get-AzVmssVMRunCommand' no longer supports the parameter 'SubscriptionId' and no alias was found for the original parameter name.","Add the parameter 'SubscriptionId' back to the cmdlet 'Get-AzVmssVMRunCommand', or add an alias to the original parameter name."
```

Comment:
this is breaking change

Reply:
wrote in the description above, but that parameter set was not working to begin with

Reply:
please try set VirtualMachineScaleSetVMRunCommandsClient.SubscriptionId
before this line: 
https://github.com/Azure/azure-powershell/blob/a81b266b8a4e0f7e39b2a8cc164e3895e8f85634/src/Compute/Compute/VirtualMachine/RunCommand/GetAzVmssVmRunCommand.cs#L75